docs/docs/working-with-Etsy-in-Gatsby.md

```diff
@@ -0,0 +1,59 @@
+---
+Title: Guide on working with Etsy in Gatsby
+---
+
+# Guide on working with Etsy in Gatsby
+
+## Etsy in Gatsby
+
+This guide will walk you through the steps needed in order to successfully work with Etsy e-commerce sites in Gatsby.
```

Comment:
"steps needed" sounds more like a tutorial than a reference guide. Take a look at the [reference guide templates](https://www.gatsbyjs.org/contributing/docs-templates/#reference-guides) and make sure this document is aligned with those recommendations.

Reply:
This was marked as resolved, but I don't see changes yet–@bball07 are you updating the PR to work more as a Reference Guide?